code/modules/food_and_drinks/machinery/processor.dm

```diff
@@ -213,27 +214,28 @@
 				break
 	if(!picked_slime)
 		return
-	var/datum/food_processor_process/P = PROCESSOR_SELECT_RECIPE(picked_slime)
-	if (!P)
+	var/datum/food_processor_process/recipe = PROCESSOR_SELECT_RECIPE(picked_slime)
+	if (!recipe)
 		return
 
 	visible_message(span_notice("[picked_slime] is sucked into [src]."))
 	LAZYADD(processor_contents, picked_slime)
 	picked_slime.forceMove(src)
 
 /obj/machinery/processor/slime/process_food(datum/food_processor_process/recipe, atom/movable/what)
-	var/mob/living/simple_animal/slime/S = what
-	if (istype(S))
-		var/C = S.cores
-		if(S.stat != DEAD)
-			LAZYREMOVE(processor_contents, S)
-			S.forceMove(drop_location())
-			S.visible_message(span_notice("[C] crawls free of the processor!"))
-			return
-		for(var/i in 1 to (C+rating_amount-1))
-			var/atom/movable/item = new S.coretype(drop_location())
-			adjust_item_drop_location(item)
-			SSblackbox.record_feedback("tally", "slime_core_harvested", 1, S.colour)
-	..()
+	var/mob/living/simple_animal/slime/processed_slime = what
+	if (!istype(processed_slime))
+		return
+
+	if(processed_slime.stat != DEAD)
+		processed_slime.forceMove(drop_location())
+		processed_slime.balloon_alert_to_viewers("crawls free!")
```

Comment:
This should be manual_emote instead

Reply:
Well maybe not it doesn't really matter

But if you wan t a balloon, remove the exclamation point

```suggestion
		processed_slime.balloon_alert_to_viewers("crawls free")
```